atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/indexing/generated/DataTable.java

```diff
@@ -12,16 +10,11 @@
 import java.util.SortedMap;
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutorService;
-import java.util.concurrent.TimeUnit;
```

Comment:
did you remove these (what I assume are) unused imports as a bulk save action or as part of a code-gen task?